testing/Jenkinsfile

```diff
@@ -0,0 +1,51 @@
+// Will run on the agent with name or tag wvtest-AuctionDriver
+node('wvtest-AuctionDriver') {
+    try {
+	// Checkout command for Jenkinsfile
+	checkout scm
+
+        // Perform routine maintenance tasks since AuctionDriver is a persistent VM
+        stage('maintain driver') {
+        // Remove unneeded docker images which take up lots of space on the AuctionDriver. These accumulate after each build.
+        // docker rmi typically returns a code of 1 because it's unable to delete some images.
+        // Specifying '|| true' ignores the error code
+        sh label: '', script: 'docker rmi $(docker images -q) || true'
+        // may also need to zip, delete, or archive large/unnecessary run results files
+		// Remove any stopped containers which might have been left over from testing
+        sh label: '', script: 'docker container prune -f'
+        }
+        stage('checkout') {
+            // Checkout command for Jenkins UI
+            //checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/vmware/weathervane']]])
+        }
+        stage('build') {
+            
+            // Pull openjdk:8 image from Harbor Proxy Cache; will default to DockerHub otherwise and probably fail
+            sh label: '', script: 'docker pull harbor-repo.vmware.com/dockerhub-proxy-cache/library/openjdk:8' 
+            sh label: '', script: 'docker tag harbor-repo.vmware.com/dockerhub-proxy-cache/library/openjdk:8 openjdk:8'
+            
+            withCredentials([usernamePassword(credentialsId: 'robot-credentials', passwordVariable: 'ROBOT_PASSWORD', usernameVariable: 'ROBOT_USERNAME')]) {
+                // Build executables and images, push to Harbor
+				sh label: '', script: 'printf \'$ROBOT_PASSWORD\' | ./buildDockerImages.pl --private --host \'harbor-repo.vmware.com/weathervaneci\' --username $ROBOT_USERNAME'
```

Comment:
I'm wondering if this harbor repository name should also be a variable?  Although it is also in the config files.

Reply:
I agree it should be a variable, so that if we want to change the account name, all the pipelines (released, 2.0.10-dev and my own feature branch) can continue working. I'll keep that in mind for my next revision.